docs/charon/charon_cli_reference.md

```diff
@@ -183,36 +194,81 @@ Usage:
+      --log-format string              Log format; console, logfmt or json (default "console")
+      --log-level string               Log level; debug, info, warn or error (default "info")
+      --loki-addresses strings         Enables sending of logfmt structured logs to these Loki log aggregation server addresses. This is in addition to normal stderr logs.
+      --loki-service string            Service label sent with logs to Loki. (default "charon")
+      --monitoring-address string      Listening address (ip and port) for the prometheus and pprof monitoring http server. (default "127.0.0.1:3620")
+      --p2p-allowlist string           Comma-separated list of CIDR subnets for allowing only certain peer connections. Example: 192.168.0.0/16 would permit connections to peers on your local network only. The default is to accept all connections.
+      --p2p-bootnode-relay             Enables using bootnodes as libp2p circuit relays. Useful if some charon nodes are not publicly accessible. Deprecated, should always be enabled. (default true)
+      --p2p-bootnodes strings          Comma-separated list of discv5 bootnode URLs or ENRs. Deprecated, use p2p-relays flag.
+      --p2p-bootnodes-from-lockfile    Enables using cluster lock ENRs as discv5 bootnodes. Allows skipping explicit bootnodes if key generation ceremony included correct IPs. Discv5 is deprecated, use relay discovery.
+      --p2p-denylist string            Comma-separated list of CIDR subnets for disallowing certain peer connections. Example: 192.168.0.0/16 would disallow connections to peers on your local network. The default is to accept all connections.
+      --p2p-disable-reuseport          Disables TCP port reuse for outgoing libp2p connections.
+      --p2p-external-hostname string   The DNS hostname advertised by libp2p. This may be used to advertise an external DNS.
+      --p2p-external-ip string         The IP address advertised by libp2p. This may be used to advertise an external IP.
+      --p2p-max-connections int        Libp2p maximum number of peers that can connect to this relay. (default 16384)
+      --p2p-max-reservations int       Updates max circuit reservations per peer (each valid for 30min) (default 512)
+      --p2p-relay-loglevel string      Libp2p circuit relay log level. E.g., debug, info, warn, error.
+      --p2p-relays strings             Comma-separated list of libp2p relay URLs or ENRs. (default [http://bootnode.lb.gcp.obol.tech:3640/enr])
+      --p2p-tcp-address strings        Comma-separated list of listening TCP addresses (ip and port) for libP2P traffic. Empty default doesn't bind to local port therefore only supports outgoing connections.
+      --p2p-udp-address string         Listening UDP address (ip and port) for discv5 discovery. Empty default disables discv5 discovery. Discv5 is deprecated, use relay discovery.
+
 ```
 
-## Host a bootnode
+## Host a bootnode (Deprecated)
```

Comment:
Deprecated section. Needs to be deleted in future releases when charon ends support for `charon bootnode`

Reply:
Can we make that a lowercase D